mno_analysis_tools/window_of_downtime.py

```diff
@@ -0,0 +1,70 @@
+# Compute maximum window of time with 0 messages
```

Comment:
Can you rename this script to give at an "action" title.

For example, "fetch_archives" was a good title because it was very clear what the high-level action that script was going to make was. If you had called it "archives" it would have been much less clear - are you downloading, creating, deleting etc.? "window_of_downtime" is more like "archives" than "fetch_archives" at the moment.

Reply:
There is already a file with that name